lazy_tensor_core/lazy_bench.py

```diff
@@ -449,48 +483,84 @@ def run_tracing_execute_noops(test, lazy_benchmark):
+            lazy_benchmark = benchmark_cls(device='lazy', jit=False)
+            # TODO: might be redundant
+            gc.collect()
+        
+            current_name = name
+            current_device = device
+
+            if device == 'cuda':
+                assert 'LTC_TS_CUDA' in os.environ and bool(os.environ['LTC_TS_CUDA'])
+
+            if args.run_tracing_execute_noops:
+                print(f"Profiling {name}")
+                run_tracing_execute_noops(args.test, lazy_benchmark)
+                # when profiling, we really don't want to do anything else
+                exit(0)
+
+            with pick_grad(args, name):
+
+                try:
+                    if args.test == 'eval':
+                        # Correctness Check
+                        torch.manual_seed(1337)
+                        model, example_inputs = benchmark.get_module()
+                        model.eval()
+                        correct_result = call_model_with(model, example_inputs)
+                        torch.manual_seed(1337)
+                        lazy_model, lazy_inputs = lazy_benchmark.get_module()
+                        lazy_model.eval()
+                        lazy_result = call_model_with(lazy_model, lazy_inputs)
+                        if not check_results(name, correct_result, lazy_result, device):
+                            print(f"INCORRECT ({name})")
+                            continue
+                except Exception:
+                    logging.exception("unhandled error")
+                    print(f"ERROR ({name})")
+                    continue
+                lazy_overhead_experiment(args, results, benchmark, lazy_benchmark)
+
+                with fuser(args.fuser) if args.fuser != 'noopt' else optimized_execution(False):
+                    if args.fuser == 'noopt':
+                        # TODO(whc) cleaner way to configure the fusers; seems i have to set both optimized_execution(False)
+                        # _and_ disable fusers to get no-optimization
+                        torch._C._jit_override_can_fuse_on_cpu(False)
+                        torch._C._jit_override_can_fuse_on_gpu(False)
+                        torch._C._jit_set_texpr_fuser_enabled(False)
+                        torch._C._jit_set_nvfuser_enabled(False)
+
+                    # using LazySync
+                    lazy_compute_experiment(args, f"amortized {args.inner_loop_repeat}x", results, benchmark, lazy_benchmark)
+                    lazy_compute_experiment(args, "unamortized", results, benchmark, lazy_benchmark, sync_every_iter=True)
+            
+        exit(0)
+
+    import subprocess
+    for model_name in iter_models(args):
+        # if `--run_in_subprocess` is specified, it will override any filters and excludes
+        # pass the rest of arguments intact such as device, test, repeat, etc
+        launch_command = f"python {' '.join(copy_argv)} --run_in_subprocess '{model_name}'"
```

Comment:
sigh.. and then need to add argparse support for --suprocess_output_file so you can pass it in...